app/javascript/react/components/capes/CapeShowContainer.js

```diff
@@ -58,8 +59,13 @@ const CapeShowContainer = (props) => {
         newReviewBody.review
       ])
     })
-    .catch(error => console.error(`Error in fetch: ${error.message}`))
-  }
+    .catch(error => {
+      if (error.message === "401 (Unauthorized)"){
+        setLoginError("Signup or Log In to review this hero")
+      }
+      console.error(`Error in fetch: ${error.message}`);
+    })
+}
```

Comment:
if the user is not logged in, they shouldnt be given the option of filling out the form at all, so you shouldnt need this error handling